server/controllers/inconsistentUsers.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Should this file be put in the `controllers` directory or would it be better to create a separate `data` directory perhaps for storing this? What do you think?

Reply:
For now, I think we can put it here, if in the future the codebase scales to a significant level, we can move it in the project root as an independent file.